### ISSUE TITLE
What is test_predict function in Machine Learning Lab?

### ISSUE THREAD

**Issue by annegentle** [CiscoDevNet/learning-labs-issues#78]
From DevNet Developer Support Webex Teams room 1/23/19:

Martin Isaksen5:38 PM
For the Machine Learning python Devnet training in step 5 - https://learninglabs.cisco.com/lab/machine-learning-101/step/5       test_predict is not defined and generates an error.  How should test_predict be defined?  (very last step in the python script)

Kassandra Keeton6:36 PM
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html <---- thought this would help, but without knowing what classified was used to return information hard to say ... likely it's one of the other labels. Maybe it's supposed to be test_data?

**Comment by annegentle** [comment]
Patrick may be able to figure this out on his own or with the owner of the lab. Let's keep this open for now.

**Comment by annegentle** [comment]
Closing as that Lab has been retired.
